R/01-etl.R

```diff
@@ -53,8 +53,10 @@ report.default <- function(x, ...) {
   updated <- data.frame(
     tableName = x$name,
     completed = as.character(Sys.time()),
+    # hard-coded: currently mappingIds exist already on the DB but not yet on the beta branch
+    mappingId = "IsoMemo",
     nrow = nrow(x$dat),
```

Comment:
@jan-abel-inwt Could you check if the column names match to those on the server? Thanks!